workspaces/desktop/config/xfce4/xfconf/xfce-perchannel-xml/xfce4-desktop.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<channel name="xfce4-desktop" version="1.0">
+  <property name="backdrop" type="empty">
+    <property name="screen0" type="empty">
+      <property name="monitorscreen" type="empty">
+        <property name="workspace0" type="empty">
+          <property name="color-style" type="int" value="0"/>
+          <property name="image-style" type="int" value="5"/>
+          <property name="last-image" type="string" value="/home/prairielearner/.local/wallpaper.jpg"/>
```

Comment:
Is this wallpaper image freely licensed?

Reply:
It’s stolen from https://publicaffairs.illinois.edu/resources/zoom-backgrounds/, so I have no idea?  I can find a CC or similar replacement for it.

Reply:
It's so annoying when people say "here's some content" but fail to license it. I think we should probably find a different image, sadly.

I'm actually also wondering about whether we'd be better off using a plain color background to save on bandwidth? I think VNC has to push chunks of the background to the client every time we drag a window and reveal new bits of the background, and a highly-textured image like this one won't compress well. I could easily be overthinking this, however, and maybe a nice background isn't much more expensive? I love the look of a nice background.